pf4j/src/main/java/ro/fortsoft/pf4j/PluginWrapper.java

```diff
@@ -54,7 +54,7 @@ public String getPluginPath() {
 	 * for this plug-in. The class loader can be used to directly access
 	 * plug-in resources and classes.
 	 */
-    public PluginClassLoader getPluginClassLoader() {
+    public ClassLoader getPluginClassLoader() {
```

Comment:
Another thing the PluginClassLoader doesn't expose any extra API only the ClassLoader public methods so I don't see a big drawback here.